File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int count_letters(text);
int main(void)
{
string text = get_string("text: ");
//count the number of letters
int letters = count_letters(text);
printf("%d\n",letters);

}
int count_letters(string text)
{
int count=0;
int length = strlen(text);
for(int i=0;i<length;i++)
{
    if(isalpha(i))
    {
      count++;
    }
    else
    {
     continue;
    }
}
return count;
}

File: sentimental-readability/readability.py
# TODO
from cs50 import get_string
def main():
    text = get_string("text: ")
    print(count_letters(text))

def count_letters(text):
    count = 0
    for c in text:
        if c.isalpha():
            count = count + 1
        else:
            continue
    return count

#def count_words(text):
#def count_sentences(text):


if __name__ == "__main__":
    main()